### ISSUE TITLE
Range validator cannot be used on Optional fields

### ISSUE THREAD

**Issue by Leopard2A5** [Keats/validator#23]
```rust
#[macro_use]
extern crate validator_derive;
extern crate validator;

use validator::Validate;

#[derive(Validate)]
pub struct Foo {
	#[validate(length(min = "1"))]
	pub name: String,
	#[validate(range(min = 0, max = 120))]
	pub age: Option<i32>,
}
```

yields 

> Invalid attribute #[validate] on field `age`: Validator `range` can only be used on number types but found `Option<i32>`

Using Validator 0.4.0

**Comment by Keats** [comment]
That's because I'm silly: https://github.com/Keats/validator/blob/master/validator_derive/src/lib.rs#L21-L23

Notice anything missing in the Option brackets...?

**Comment by Leopard2A5** [comment]
Who cares about signedness? :smile:

**Comment by Keats** [comment]
Thanks for the PR. It's fixed in 0.5.0 of validator_derive which also bumps serde to 1.0
